src/editor/nodes/TroikaTextNode.js

```diff
@@ -0,0 +1,219 @@
+// This is a port of https://github.com/protectwise/troika/tree/master/packages/troika-three-text
+// that has been made functional and restyled for Mozilla Hubs and Spoke (hubs.mozilla.com)
+// by @jamesckane at Paradowski Creative (paradowski.com)
+
+import { Color, Object3D } from "three";
+import EditorNodeMixin from "../../../../src/editor/nodes/EditorNodeMixin";
+import { Text } from "troika-three-text";
+
+export default class TroikaTextNode extends EditorNodeMixin(Object3D) {
```

Comment:
Maybe it we could just inherit from Text directly instead of Object3D and save the extra node?